bool isHover
bool isHovering
bool isLanding
---
